izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanelAutomationHelper.java

```diff
@@ -44,13 +44,13 @@ public UserPathPanelAutomationHelper()
     public void createInstallationRecord(InstallData idata, IXMLElement panelRoot)
     {
         // Installation path markup
-        IXMLElement ipath = new XMLElementImpl(UserPathPanel.pathElementName, panelRoot);
+        IXMLElement ipath = new XMLElementImpl(UserPathPanel.PATH_ELEMENT_NAME, panelRoot);
```

Comment:
Why not using a static import for this constant here and for the following usages?

Reply:
Done, using static import now.